modules/op-release-notes-1-2.adoc

```diff
@@ -13,8 +13,8 @@
 * Tekton `tkn` CLI 0.13.1
 * Tekton Triggers 0.8.1
 * ClusterTasks based on Tekton Catalog 0.16
-* IBM Power Systems on {product-title} 4.6
-* IBM Z and LinuxONE on {product-title} 4.6
+* IBM Power Systems on {product-title} 4.7
```

Comment:
I'd just double check this with the pipelines team

Reply:
@bergerhoffer Do you have a contact for that team? I can't seem to find one. Thanks!

Reply:
@Preeticp Should be a good contact. @Preeticp can you confirm whether this is okay to change from 4.6 to 4.7 (for the OCP 4.7 GA)

Reply:
I think we should not do this change, as this section lists what is supported by Pipelines TP 1.2. IIUC the upcoming TP 3 release would support the 4.7 versions. I will ask the team and confirm on this.

Reply:
I have confirmation from @SNiemann15 that we might not have support for 4.7 in TP 1.3 too. So, let's not go ahead with the changes on L16 and L17, please.

Reply:
@Preeticp Thank you! I am reverting this change.